index.d.ts

```diff
@@ -78,6 +78,15 @@ export default class QuickLRU<KeyType, ValueType> extends Map<KeyType, ValueType
 	*/
 	peek(key: KeyType): ValueType | undefined;
 
+	/**
+	Get the remaining time to live for an item, or undefined for item not in cache.
+	Does not mark it as recently used.
+	Does not remove entry from cache when item is expired.
+
+	@returns Remaining time to live in ms when set, or infinity when maxAge is not set. Undefined when item does not exist.
+	*/
+	ttl(key: KeyType): number | undefined;
```

Comment:
```suggestion
	expiresIn(key: KeyType): number | undefined;
```

I prefer this. It goes better with the `maxAge` option naming.

Reply:
ok, done!